<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XPlay</class>
 <widget class="QWidget" name="XPlay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XPlay</string>
  </property>
  <widget class="XPlayOpenGLWidget" name="openGLWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(239, 41, 41)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>530</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>530</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>play</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalSecs">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">fontcolor:rgb(239, 41, 41)</string>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="playTime">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>530</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>000:000</string>
   </property>
  </widget>
  <widget class="QLabel" name="sp">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>530</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QSlider" name="playProgress">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>731</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XPlayOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>xplayopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
